pkg/instrumentation/sdk.go

```diff
@@ -171,6 +171,18 @@ func (i *sdkInjector) injectCommonSDKConfig(ctx context.Context, otelinst v1alph
 			Value: chooseServiceName(pod, resourceMap, appIndex),
 		})
 	}
+
+	idx = getIndexOfEnv(container.Env, constants.EnvOTELServiceVersion)
+	if idx == -1 {
+		vsn := chooseServiceVersion(pod, appIndex)
+		if vsn != "" {
+			container.Env = append(container.Env, corev1.EnvVar{
```

Comment:
@KitSutliff have you reviewed the telemetry produced by a service that had this env var and the auto-instrumentation injected?  While there is an [SDK requirement to support OTEL_SERVICE_NAME](https://opentelemetry.io/docs/specs/otel/sdk-environment-variables/#general-sdk-configuration), there isn't a requirement for `OTEL_SERVICE_VERSION`.  To get the SDK to emit `service.version` I believe the operator needs to update the `OTEL_RESOURCE_ATTRIBUTES` env var and include `service.version`.  In this function it would be the `resourceMap`.